vendor/github.com/coreos/go-oidc/v3/oidc/oidc.go

```diff
@@ -192,6 +224,16 @@ type UserInfo struct {
 	claims []byte
 }
 
+type userInfoRaw struct {
+	Subject string `json:"sub"`
+	Profile string `json:"profile"`
+	Email   string `json:"email"`
+	// Handle providers that return email_verified as a string
+	// https://forums.aws.amazon.com/thread.jspa?messageID=949441&#949441 and
+	// https://discuss.elastic.co/t/openid-error-after-authenticating-against-aws-cognito/206018/11
+	EmailVerified stringAsBool `json:"email_verified"`
```

Comment:
🤯

Reply:
I guess we're not making use of the UserInfo fetching at all... still not a huge fan of this relaxing this way